src/services/info/notice/notice-detail/NoticeDetailPage.vue

```diff
@@ -62,6 +64,20 @@
             </p-button>
         </section>
         <!--song-lang-->
+        <p-button-modal :header-title="$t('Send the mail')"
+                        :visible.sync="sendEmailModalVisible"
+                        :size="'sm'"
```

Comment:
props를 넘길 때, 앞에 v-bind:( or `:`)를 작성하는 경우 값에는 코드처럼 작성해야 하다보니 문자열을 표현하기위해 `'`가 붙었네요!
```suggestion
                        size="sm"
```
요롷게 사용하시면 "" 의 값을 string으로 사용할 수 있어요!

Reply:
아하 그렇군여 감사합니다!